src/webapp/pages/form-page/disease-outbreak-event/utils/updateDiseaseOutbreakEventFormState.ts

```diff
@@ -18,16 +19,19 @@ export function updateDiseaseOutbreakEventFormState(
     currentUserUsername: string
 ): FormState {
     const updatedForm = updateFormStateAndApplySideEffects(prevFormState, updatedField);
+    const updatedFormWithRulesApplied = diseaseOutbreakEventWithOptions.rules.length
```

Comment:
can we add a condition to apply rule only if rule corresponding to updated field exists?